CreeDictionary/API/models.py

```diff
@@ -376,7 +379,7 @@ def fetch_lemma_by_user_query(user_query: str, **kwargs) -> "CreeAndEnglish":
             ids_by_suffix = Wordform.affix_searcher.search_by_suffix(user_query)
 
             for wf in Wordform.objects.filter(
-                id__in=set(chain(ids_by_prefix, ids_by_suffix))
+                id__in=set(chain(ids_by_prefix, ids_by_suffix)), **extra_constraints
```

Comment:
`**extra_constraints` was missed here --- it should be included inside every Queryset.filter call.